roles/openshift_metrics/README.md

```diff
@@ -90,9 +90,10 @@ Example Playbook
 
 ```
 - name: Configure openshift-metrics
-  hosts: oo_first_master
+  hosts: oo_masters_to_config
   roles:
   - role: openshift_metrics
+  run_once: True
```

Comment:
I'm not a fan of using run_once at the play level as shown in the example in the README.  The run_once will be applied to **every task** in the entire play (and role tasks) which may not be desired outcome.  However, including run_once at the Play level superceeds the need to add it to each task in the role task file as shown.  Only the task which needs run_once: False should be updated.  run_once: True should be either at the Play/Role level or at the task (or include) level.

So, I don't like the idea of a play/role needing to be called with a run_once in order for it to operate properly.